packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Spinner, SpinnerSize, Stack } from '@fluentui/react';
+import React from 'react';
+import { useLocale } from '../../localization';
+import { StreamMedia } from '../StreamMedia';
+import { VideoTile } from '../VideoTile';
+import { VideoStreamOptions, VideoGalleryRemoteParticipant } from '../../types';
+import { videoWithNoRoundedBorderStyle, loadingStyle } from './styles/RemoteScreenShare.styles';
+import { _formatString } from '@internal/acs-ui-common';
+
+/**
+ * All strings that may be shown on the UI in the {@link RemoteScreenShare}.
+ *
+ * @public
+ */
+export interface RemoteScreenShareStrings {
+  /** String to show when remote screen share stream is loading */
+  screenShareLoadingMessage: string;
+}
+
+/**
+ * A memoized version of VideoTile for rendering the remote screen share stream. React.memo is used for a performance
+ * boost by memoizing the same rendered component to avoid rerendering this when the parent component rerenders.
+ * https://reactjs.org/docs/react-api.html#reactmemo
+ */
+export const RemoteScreenShare = React.memo(
+  (props: {
+    screenShareParticipant?: VideoGalleryRemoteParticipant;
+    onCreateRemoteStreamView?: (userId: string, options?: VideoStreamOptions) => Promise<void>;
+  }) => {
+    const { onCreateRemoteStreamView, screenShareParticipant } = props;
+    const locale = useLocale();
+
+    const screenShareStream = screenShareParticipant?.screenShareStream;
+    const videoStream = screenShareParticipant?.videoStream;
+    if (screenShareStream?.isAvailable && !screenShareStream?.renderElement) {
```

Comment:
Also what is the difference between this `if` and the check right below this? They both have the same if body:
`screenShareParticipant && onCreateRemoteStreamView && onCreateRemoteStreamView(screenShareParticipant.userId);`

Reply:
The second `if` statement was unnecessary removed. As for using `useEffect`, I don't think it is needed because the stream can only change when the `screenShareParticipant` changes which is a prop anyways.